query/src/servers/mysql/mysql_interactive_worker.rs

```diff
@@ -48,9 +50,34 @@ use crate::sessions::QueryContext;
 use crate::sessions::SessionRef;
 use crate::sessions::TableContext;
 use crate::sql::DfParser;
+use crate::sql::DfStatement;
 use crate::sql::PlanParser;
 use crate::sql::Planner;
 
+fn is_result_set(stmt: &DfStatement) -> bool {
```

Comment:
We will remove old statements in the future, I think we should match new statements/planners to have `is_result_set`